### ISSUE TITLE
Creating a ChronicleMap or a ChronicleMapBuilder takes over 10 seconds

### ISSUE THREAD

**Issue by xjru** [OpenHFT/Chronicle-Core#16]
Creating a ChronicleMap or a ChronicleMapBuilder for the first time takes over 10 seconds. Here's the code:
```
ChronicleMapBuilder<String, String> builder =
        ChronicleMapBuilder.of(String.class, String.class)
        .averageKey("999234567")
        .averageValue("999234567")
        .entries(10);

ChronicleMap<String, String> coll = builder.create();
```
The 10 second delay happens on the first statement. I ran the code in the profiler. Here's what I found:
```
net.openhft.chronicle.core.OS.getProcessId0()	44.318596	5,076 ms (44.3%)	5,076 ms	1
net.openhft.chronicle.core.OS.getHostName0()	44.287758	5,072 ms (44.3%)	5,072 ms	1
```
Creating a second map or builder in the same program causes no delay at all.

I'm using chronicle-map 3.10.1 from maven on macOS 10.12. My Java version is Java(TM) SE Runtime Environment (build 1.8.0_91-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.91-b14, mixed mode)

**Comment by peter-lawrey** [comment]
These calls are getting the host name.  If you don't have DNS reverse
lookup or an /etc/hosts entry setup correctly they can take a long time.
This is not something you can fix from Java.

**Comment by xjru** [comment]
Thanks Peter!

You're right that this can be fixed in /etc/hosts by adding the actual computer name as shown in Preferences->Sharing, including the .local suffix.

The problem with adding the computer name to /etc/hosts is that computer names on macOS keep changing automatically. I'm at name-3 right now so I added name-3.local to /etc/hosts for both 127.0.0.1 and ::1. Once the name gets bumped to name-4, everything will break again. 

This is clearly a macOS and/or Java issue and has nothing to do with Chronicle. Something seems to have changed in macOS Sierra about the way in which java.net.InetAddress.getLocalHost() works. I found many reports on the same issue related to other Java based software such as Tomcat and sbt.
